packages/block-library/src/cover/save.js

```diff
@@ -56,7 +57,9 @@ export default function save( { attributes } ) {
 	if ( customGradient && ! url ) {
 		style.background = customGradient;
 	}
-	style.minHeight = minHeight || undefined;
+	style.minHeight = minHeightUnit
+		? `${ minHeight }${ minHeightUnit }`
+		: minHeight || undefined;
```

Comment:
I for one had to go check the [operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence) table for logical OR and ternary conditional to make sure I was reading this correctly. :) I recommend parentheses in this situations.